account_manager/src/validator/exit.rs

```diff
@@ -0,0 +1,440 @@
+use crate::wallet::create::STDIN_INPUTS_FLAG;
+use crate::{SECRETS_DIR_FLAG, VALIDATOR_DIR_FLAG};
+use bls::{Keypair, PublicKey};
+use clap::{App, Arg, ArgMatches};
+use directory::{parse_path_or_default_with_flag, DEFAULT_SECRET_DIR};
+use environment::Environment;
+use eth2::{
+    types::{GenesisData, StateId, ValidatorId, ValidatorStatus},
+    BeaconNodeHttpClient, Url,
+};
+use eth2_testnet_config::Eth2TestnetConfig;
+use safe_arith::{ArithError, SafeArith};
+use std::path::PathBuf;
+use std::time::{SystemTime, UNIX_EPOCH};
+use types::{ChainSpec, Epoch, EthSpec, Fork, Hash256, Slot, VoluntaryExit};
+use validator_dir::{Manager as ValidatorManager, ValidatorDir};
+
+pub const CMD: &str = "exit";
+pub const VALIDATOR_FLAG: &str = "validator";
+pub const BEACON_SERVER_FLAG: &str = "beacon-node";
+pub const PASSWORD_PROMPT: &str = "Enter the keystore password";
+
+pub const DEFAULT_BEACON_NODE: &str = "http://localhost:5052/";
+pub const CONFIRMATION_PHRASE: &str = "Exit my validator";
+pub const WEBSITE_URL: &str = "https://lighthouse-book.sigmaprime.io/voluntary-exit.html";
+pub const PROMPT: &str =
+    "WARNING: WITHDRAWING STAKED ETH WILL NOT BE POSSIBLE UNTIL ETH1/ETH2 MERGE";
```

Comment:
```suggestion
    "WARNING: WITHDRAWING STAKED ETH IS NOT PRESENTLY POSSIBLE";
```

Just to avoid committing to a time when withdraws will be permitted, since it's out of our control.